lib/mustache/context.rb

```diff
@@ -138,8 +138,11 @@ def fetch(name, default = :__raise)
     def find(obj, key, default = nil)
       return find_in_hash(obj.to_hash, key, default) if obj.respond_to?(:to_hash)
 
-      key = to_tag(key)
-      return default unless obj.respond_to?(key)
+      if !obj.respond_to?(key)
+        # no match for the key, but it may include a hyphen, so try again replacing hyphens with underscores.
+        key = key.to_s.tr('-', '_')
+        return default if !obj.respond_to?(key)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Favor `unless` over `if` for negative conditions.](https://www.codacy.com/app/rokusu/mustache/file/1992713325/issues/source?bid=2903613&fileBranchId=3119888#l144)